### PR TITLE
Refactor disable-service-access command

### PR DESCRIPTION
**Pull request by nmaslarski**
This commit refactors the `cf disable-service-access` command to the new
style, and adds integration tests.

The changes are detailed in full here:
https://www.pivotaltracker.com/story/show/161378857

### CODE REVIEW

**Comment by nmaslarski** [comment]
Hay @h0nIg,
Sorry but no it won't fix it.

**Comment by h0nIg** [comment]
@nmaslarski any chance to fix the issue with this pullrequest?

**Comment by nmaslarski** [comment]
Normally we don't want to add new behaviour with the refactors, but if it's a bug I'm not sure
@laurzca @mattmcneeney @abbyachau

**Comment by mattmcneeney** [comment]
I think #1530 is a bug and should be fixed in a separate PR

File: actor/v2action/service_access.go [hunk 1]
```diff
@@ -100,14 +100,126 @@ func (actor Actor) EnablePlanForOrg(serviceName, servicePlanName, orgName string
...
+
+// DisablePlanForAllOrgs disables access to a specific plan of the given service in all orgs.
+func (actor Actor) DisablePlanForAllOrgs(serviceName, servicePlanName string) (Warnings, error) {
+	servicePlans, allWarnings, err := actor.GetServicePlansForService(serviceName)
+	if err != nil {
+		return allWarnings, err
+	}
+
+	for _, plan := range servicePlans {
+		if plan.Name == servicePlanName {
```
[the remark this answers is not in the document]
Comment (AartiKriplani) [reply actor/v2action/service_access.go]: According to the [OSBAPI spec](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#service-plan-object), `name MUST be unique within the Service Offering`, so we wouldn't have that issue here

**Pull request merged**